tests/testthat/tests.R

```diff
@@ -665,6 +665,29 @@ test_that("tile colorRamp2 palette",{
 	
 })
 
+test_that("annotation tile factor colour order",{
+  
+  # If factor levels correctly interpreted, colour palette should read in order: 
+  # "first_level", "second_level", "third_level", "fourth_level"
+
+  p = 
```

Comment:
Can you please add another test where you manipulate the factor order 

```r
levels = c( "second_level", "third_level", "fourth_level", "first_level")
```

And you test the effect